spark/pyspark_filter_wind_csv.py

```diff
@@ -0,0 +1,23 @@
+# This program takes an input csv file "longspeak_wtk.csv" and outputs a filtered dataset
+
+# Access the SparkSession module from pyspark
+from pyspark.sql import SparkSession
+import csv
+
+
+# Open a Spark session
+spark = SparkSession.builder.appName("find_the_wind").getOrCreate()
+       
+df = spark.read.csv("s3a://wind-toolkit-hdf5/longspeak_wtk.csv")
+df = spark.read.format('csv').options(header='true', inferSchema='true').load("s3a://wind-toolkit-hdf5/longspeak_wtk.csv")
+df = spark.read.format('csv').options(header='true', inferSchema='true').load('test_data/longspeak_wtk.csv')
+
+
+df.show(5)
```

Comment:
right now this is printing rather than saving...final code should always be saving to desired output format, not printing